### DISCUSSION TITLE
READMEのコマンド修正依頼

### DISCUSSION THREAD

**Discussion by kryota-dev** [increments/qiita-discussions#550] [要望・アイデア・不具合]
該当箇所は[こちら](https://github.com/increments/qiita-cli#--credential-credential_dir)

以下のコマンドに誤りがあります（ハイフンが多い）
```shell
npx qiita login ---credential ./my_conf/
```

正しくはこちらで動作確認済みです。
```shell
npx qiita login --credential ./my_conf/
```

補足
npxではハイフン3つでも動作するのかもしれませんが（未検証）、bun（bunx）を使用する環境では修正後のコマンドでないと動作しませんでした。

**Comment by tanaka-asami415** [comment]
@ryota-k0827 
教えていただきありがとうございます🙏
修正いたしました！
